src/_posts/languages/ruby/rails/2000-01-01-start.md

```diff
@@ -131,7 +131,7 @@ but it's expected. There is nothing in the project, it's time to build your prod
 
 Latest version of Rails application also use Node.js in case ExecJS or
 Webpacker is detected. The Node.js version built inside the Ruby buildpack is
-frozen. It is 12.16.2 LTS version at the time of writing.
+frozen. It is 16.13.1 LTS version at the time of writing.
```

Comment:
Should we say `16.*` or something less specific to not have to update this doc every time ?

Reply:
Yes please 👍🏾 
```suggestion
frozen. It is the latest version of the LTS 16.x at the time of writing.
```

Reply:
Thanks for the suggestion !